examples/connext_dds/flat_data_latency/c++11/Common.hpp

```diff
@@ -18,31 +18,34 @@
 #include <dds/core/cond/GuardCondition.hpp>
 #include <dds/pub/DataWriter.hpp>
 #include <dds/pub/DataWriterListener.hpp>
+#include "application.hpp"
 
 
 /**
  * Wait for discovery
  */
-template <typename T>
+template<typename T>
```

Comment:
Not sure what the .clang-format is doing here, but we do add a space between `template` and `<`.

If .clang-format is forcing you to remove it, that's fine.

Reply:
Yes, clang-format is forcing to remove it.